### PR TITLE
chore: update change logs for e4.4.21

### PR DESCRIPTION
**Pull request by terry-xiaoyu**

### CODE REVIEW

File: zh_CN/changes/changes-ee-4.4.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+- 添加了对 Confluent 数据桥的支持。
+
+- 现在 Kafka 消费组里的 MQTT 主题字段支持包含占位符的模板了。
+
+  比如 Kafka 消费到消息里 key 为 "a"，配置的 MQTT 主题为 "topic/${key}"，那么在消息转发的是，MQTT 主题会被替换为 "topic/a"。
+
+- 现在 “消息重发布” 动作支持在添加 MQTT 属性和用户属性了。
+
+  “消息重发布” 动作支持了两个新的字段："MQTT Properties" 和 "User Property Value"，都是键值对的格式，并且键和值都支持占位符。
```
Comment (lina50) [inline zh_CN/changes/changes-ee-4.4.md]: MQTT Properties -> mqtt_properties_template
 User Property Value -> user_properties_template   ？

File: en_US/changes/changes-ee-4.4.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
+- Added support for Confluent data bridge.
+
+- Now the MQTT topic field in the Kafka consumer group supports templates with placeholders.
+
+  For example, if the key of the message consumed by Kafka is "a", and the configured MQTT topic is "topic/${key}", then when the message is forwarded, the MQTT topic will be replaced with "topic/a".
+
+- Now the "Message Republish" action supports adding MQTT properties and user properties.
+
+  The "Message Republish" action supports two new fields: "MQTT Properties" and "User Property Value", both of which are in the format of key-value pairs, and both the key and value support placeholders.
```
Comment (lina50) [inline en_US/changes/changes-ee-4.4.md]: image
MQTT Properties -> mqtt_properties_template

User Property Value -> user_properties_template

File: en_US/changes/changes-ee-4.4.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+
+### Bug fixes
+
+- Fixed the issue that the Kafka action cannot send numeric values as Kafka Headers.
+
+  Prior to this fix, when the "Kafka Headers value encode mode" is set to "NONE", if the "Kafka Headers" field contains a JSON object with numeric types (such as `{"a": 1, "b": "str"}`), the numeric values (`"a":1`) will be ignored and not sent to Kafka. After the fix, the numeric types in JSON will be converted to strings before being sent to Kafka.
```
Comment (lina50) [inline en_US/changes/changes-ee-4.4.md]: strings -> string?

File: en_US/changes/changes-ee-4.4.md (hunk 2 above)
Comment (terry-xiaoyu) [reply en_US/changes/changes-ee-4.4.md]: Thanks! Now you are not just a technical writer but are an emqx expert!

File: zh_CN/changes/changes-ee-4.4.md (hunk 1 above)
Comment (terry-xiaoyu) [reply zh_CN/changes/changes-ee-4.4.md]: The `mqtt_properties_template` is invisible to the user. The user can only see the "MQTT Properties" or "MQTT 属性" from the UI as field descriptions. So I will change them to the correct field description displayed on the UI.

File: en_US/changes/changes-ee-4.4.md (hunk 3 above)
Comment (terry-xiaoyu) [reply en_US/changes/changes-ee-4.4.md]: "numeric types be converted to strings. ", I am not sure if there's any grammar problems in it, I just used the `copilot`(AI) to generate the translation for me.

**Review by wivwiv — APPROVED** [review]

**Pull request merged**